easysocial/easysocial/page.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/**
+ * @package     Joomla.Site
+ * @subpackage  Com_api-plugins
+ *
+ * @copyright   Copyright (C) 2009-2014 Techjoomla, Tekdi Technologies Pvt. Ltd. All rights reserved.
+ * @license     GNU GPLv2 <http://www.gnu.org/licenses/old-licenses/gpl-2.0.html>
+ * @link        http://techjoomla.com
+ * Work derived from the original RESTful API by Techjoomla (https://github.com/techjoomla/Joomla-REST-API)
+ * and the com_api extension by Brian Edgerton (http://www.edgewebworks.com)
+ */
+
+defined('_JEXEC') or die('Restricted access');
+
+jimport('joomla.plugin.plugin');
+jimport('joomla.html.html');
+
+require_once JPATH_ADMINISTRATOR . '/components/com_easysocial/includes/foundry.php';
+require_once JPATH_ADMINISTRATOR . '/components/com_easysocial/models/pages.php';
+require_once JPATH_ADMINISTRATOR . '/components/com_easysocial/models/covers.php';
+require_once JPATH_ADMINISTRATOR . '/components/com_easysocial/models/albums.php';
+require_once JPATH_ADMINISTRATOR . '/components/com_easysocial/models/fields.php';
+require_once JPATH_SITE . '/plugins/api/easysocial/libraries/mappingHelper.php';
+require_once JPATH_SITE . '/plugins/api/easysocial/libraries/uploadHelper.php';
```

Comment:
@Pratikshakhandagale Are you using all the models or helper added here?

Reply:
@snehal0904 Removed

Reply:
@Pratikshakhandagale  - not yet removed